### PR TITLE
Sdks 1031 cicd

### PR DESCRIPTION
**Pull request by jhoongo**

### CODE REVIEW

File: FRCore/FRCoreTests/FRCore/Util/KeychainServiceTests.swift [hunk 1]
```diff
@@ -14,26 +14,39 @@ class KeychainServiceTests: FRBaseTestCase {
 
     var kc: KeychainService?
     
+    override func setUp() {
```
Comment (rodrigoareis) [inline FRCore/FRCoreTests/FRCore/Util/KeychainServiceTests.swift:17]: Update copyright

File: FRTestHost/FRTestHost/SharedTestFiles/BaseTest/FRAuthBaseTest.swift [hunk 2]
```diff
@@ -32,25 +32,13 @@ class FRAuthBaseTest: FRBaseTestCase {
     
     //  MARK: - Properties
     
-    var config: Config = Config()
```
Comment (rodrigoareis) [inline FRTestHost/FRTestHost/SharedTestFiles/BaseTest/FRAuthBaseTest.swift:35]: Update copyright

File: FRTestHost/FRTestHost/SharedTestFiles/BaseTest/FRBaseTestCase.swift [hunk 3]
```diff
@@ -15,6 +15,9 @@ import XCTest
 class FRBaseTestCase: XCTestCase {
     
     //  MARK: - Properties
+
```
Comment (rodrigoareis) [inline FRTestHost/FRTestHost/SharedTestFiles/BaseTest/FRBaseTestCase.swift:18]: Update copyright

File: FRTestHost/FRTestHost/SharedTestFiles/TestConfig/Config.swift [hunk 4]
```diff
@@ -26,6 +26,7 @@ class Config: NSObject {
     
     @objc var authServiceName: String?
     @objc var registrationServiceName: String?
+    @objc var keychainAccessGroup: String?
```
Comment (rodrigoareis) [inline FRTestHost/FRTestHost/SharedTestFiles/TestConfig/Config.swift:29]: Update copyright

**Review by rodrigoareis — CHANGES_REQUESTED** [review]
All changes looks good, just need to update the header for some changed files

**Review by spetrov — DISMISSED** [review]
Looks good to me!

**Review by george-bafaloukas-forgerock — APPROVED** [review]
Looks good

**Comment by jhoongo** [comment]
> All changes looks good, just need to update the header for some changed files

Copyright header is updated :)

**Review by rodrigoareis — APPROVED** [review]
Changes looks good to me

**Pull request merged**